share/wrf_tsin.F

```diff
@@ -24,7 +26,7 @@ SUBROUTINE wrf_tsin ( grid , ierr )
 #if ((EM_CORE == 1) && (DA_CORE != 1))
     IF ( grid%dfi_opt == DFI_NODFI .OR. (grid%dfi_opt /= DFI_NODFI .AND. grid%dfi_stage == DFI_SETUP) ) THEN
 #endif
-
+    CALL model_to_grid_config_rec ( grid%id , model_config_rec , config_flags ) 
```

Comment:
A few more columns of indentation would be nice, since we're inside of an if-test.